### ISSUE TITLE
No sound from YouTube video when iOS on silent

### ISSUE THREAD

**Issue by Loboboboboboo** [apache/cordova-plugin-inappbrowser#280]
It is normal for sound to play when playing YouTube videos in browsers or in apps when the phone is on silent - however this doesn't happen in the inappbrowser, as such I'm needing to add a warning instructing users if they hear no sound to un-mute the phone - please can there be an option to allow audio from the inappbrowser even when the phone is on silent.

**Comment by janpio** [comment]
What platform are you talking about?

**Comment by Loboboboboboo** [comment]
Phonegap - iPhone 6 - iOS 11.4.1

**Comment by janpio** [comment]
What version of Cordova are you using? What version of Cordovs-iOS? What version of the plugin?
Do you have a simple that can reproduce the behavior you describe?

**Comment by Loboboboboboo** [comment]
Is this a known thing that happens generally or is it a bug in my setup?

**Comment by janpio** [comment]
I don't know. Did you find anything in the documentation?

**Comment by Loboboboboboo** [comment]
Nope - I presumed it was just a feature that was lacking but based off your questions about my specific setup you gave me some doubts..

**Comment by janpio** [comment]
InAppBrowser is a plugin for Cordova, that enables users to show websites in another "window" in their app. This is done by creating a native container that holds a webview and some navigation UI. Maybe the webview can be configured to play sounds even when the device is on silent, and just isn't. My questions were just to understand all the specifics of the problem.

**Comment by Loboboboboboo** [comment]
Yeah, fair enough, and thanks for helping! Hopefully this can be configured some way..

**Comment by janpio** [comment]
If I had the time I would now research how this plugin creates the webview, search for some documentation on that from Apple and see if there maybe is some option to configure that.

**Comment by ternblom** [comment]
Hello,

You can fix this using this plugin [https://github.com/EddyVerbruggen/cordova-plugin-backgroundaudio](url)

I had the same problem and this plugin take care of everything.
